src/markdown.c

```diff
@@ -1413,6 +1413,25 @@ prefix_quote(uint8_t *data, size_t size)
 	return 0;
 }
 
+static size_t
+prefix_blockspoiler(uint8_t *data, size_t size)
```

Comment:
🔕  Just to make sure I'm understanding this right, what's happening here is the following?

This determines if we are in a block spoiler context, and if so, at what character position on this line it begins. It allows up to 3 space characters before the `>!` syntax which denotes a block spoiler, and up to 1 space character immediately following. If it does not find that, it will return 0, denoting that no block spoiler was found.

Kinda wish the surrounding code here was better documented.